jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpFields.java

```diff
@@ -699,7 +699,7 @@ private <T> List<HttpField> getFields(T header, BiPredicate<HttpField, T> predic
      * @return the value of the field as a {@code long},
      * or -1 if no such field is present
      * @throws NumberFormatException if the value of the field
-     * cannot be converted to a {@link long}
+     * cannot be converted to a {@link Long}
```

Comment:
Please change these to `{@code long}` since we are not really converting to `Long`.

Reply:
done